src/sas/sasgui/perspectives/fitting/fitpage.py

```diff
@@ -1732,10 +1732,7 @@ def _set_fun_box_list(self, fun_box):
         # build function (combo)box
         ind = 0
         while(ind < len(list)):
```

Comment:
Taking this one step further could eliminate the need for indexing:
`for item in list:
    fun_box.Append(item)
`

Reply:
True, although I don't know why fitpage.py and basepage.py appear here. Only the single change in model_editor.py is needed to restore the functionality for sum/multiply. The changes in fitpage and basepage are part of another branch that I created (ticket1007) which tries to solve the problem with the spherical_sld model (this is not yet finished, so no pull request yet).

Reply:
Looking at the commits, you may have accidentally branched off of your 1007 branch instead of master. The changes to fitpage and basepage should either be removed or the work for 1007 can be added here when ready.